src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```
@@ -34,14 +35,15 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+    public void addToCart(Authentication authentication, @RequestParam Long movieSessionId) {
         shoppingCartService.addSession(
-                movieSessionService.get(movieSessionId), userService.get(userId));
+                movieSessionService.get(movieSessionId),
+                userService.findByEmail(authentication.getName()).get());
```

it would be better to throw an exception with a custom error message